src/sdk/clients/createNexusClient.test.ts

```diff
@@ -157,19 +157,19 @@ describe("nexus.client", async () => {
     expect(result).toBeTruthy()
   }, 30000)
 
-  test.skip("should create a nexusAccount from an ethers signer", async () => {
-    const ethersProvider = new JsonRpcProvider(chain.rpcUrls.default.http[0])
-    const ethersSigner = new Wallet(pKey, ethersProvider)
+  // test("should create a nexusAccount from an ethers signer", async () => {
```

Comment:
Why comment instead of skip? Commenting means we have to repeatedly fix imports.